editor/linecache.go

```diff
@@ -27,69 +27,66 @@ func (lc *LineCache) addInvalid(newLines []*Line, newInvalidBefore *int, newInva
 
 func (lc *LineCache) ApplyUpdate(update *rpc.Update) {
 	// TODO: Make capacity = height of buffer
-	newLines := make([]*Line, 0, 10)
+	newLines := make([]*Line, 0, 100)
```

Comment:
this is still just a random number right?

Reply:
Oops. I did this because I thought this could be related to a problem I was having and forgot to revert it. So yes, this is just a random number. Sorry about that.